crypto/evp/evp_fetch.c

```diff
@@ -68,9 +57,9 @@ struct method_data_st {
 /*
  * Generic routines to fetch / create EVP methods with ossl_method_construct()
  */
-static void *alloc_tmp_method_store(void)
+static void *alloc_tmp_method_store(OPENSSL_CTX *ctx)
 {
-    return ossl_method_store_new();
+    return ossl_method_store_new(ctx);
```

Comment:
It took me a bit to see this myself, so I think we need to comment that `ctx` doesn't mean the allocated store ends up getting stored in in the library context but that `ctx` allows access to underlying property data needed for computation.

Reply:
Done